shared/tinyusb/tusb_config.h

```diff
@@ -2,6 +2,7 @@
  * The MIT License (MIT)
  *
  * Copyright (c) 2020-2021 Damien P. George
+ * Copyright (c) 2022 Blake W. Felt
```

Comment:
@Molorius For this file I'm not sure it's correct to add your copyright.  This file was simply moved from one location to another, so there's not much added IP.

Do you agree to the removal of this copyright line?

Reply:
Yes. If I remember correctly I initially had more functions in here which is why I added my name.